assembly/src/library/tests.rs

```diff
@@ -68,6 +68,7 @@ fn library_exports() -> Result<(), Report> {
         end
     "#;
     let bar = parse_module!(&context, "lib2::bar", bar);
+    std::println!("{bar}");
```

Comment:
I think this is probably a dubbing leftover?

Reply:
Oops, thanks